Mathlib/Analysis/VonNeumannAlgebra/Basic.lean

```diff
@@ -39,8 +39,7 @@ One the other hand, not picking one means that the weak-* topology
 (which depends on a choice of predual) must be defined using the choice,
 and we may be unhappy with the resulting opaqueness of the definition.
 -/
-class WStarAlgebra (M : Type u) [NormedRing M] [StarRing M] [CStarRing M] [Module ℂ M]
-    [NormedAlgebra ℂ M] [StarModule ℂ M] : Prop where
+class WStarAlgebra (M : Type u) [CStarAlgebra M] : Prop where
```

Comment:
This was actually a bad definition as originally written because `Module ℂ M` was a duplicated instance leading to multiple `SMul` structures on this type.